delft/secrets.nix

```diff
@@ -0,0 +1,19 @@
+let
+  keys = import ../ssh-keys.nix;
+
+  secrets = with keys; {
+    alertmanager-matrix-forwarder = [ machines.eris ];
+    fastly-read-only-api-token = [ machines.eris ];
+    packet-sd-env = [ machines.eris ];
+    prometheus-packet-spot-market-price-exporter = [ machines.eris ];
+    rfc39-credentials = [ machines.eris ];
+    rfc39-github = [ machines.eris ];
```

Comment:
Decrypting that leads to a binary file, and I don't see where this secret gets used.

Reply:
Referenced in `rfc39-credentials` :shrug: